default.yaml

```diff
@@ -133,6 +133,10 @@ ipRecording:
 
 # blacklistedIPs: ['1.2.3.4', '5.6.7.8']
 
+blockListedASNs: []
```

Comment:
could you give some example of the expected syntax here? same for countries. if we use an iso code, would could name it.